### ISSUE TITLE
[BUG] 토큰 설정 관련 버그

### ISSUE THREAD

**Issue by Seogeurim** [woowa-techcamp-2021/store-2#127]
## 개요 

request 시 토큰을 header에 setting하는 과정에서 버그가 있습니다.

## 버그 발생 단계 

로그인 후 마이페이지로 들어가 주문 목록 조회 요청을 하면 서버에 no-token 에러가 발생합니다.
하지만 로컬스토리지에는 세팅 완료입니다.

## 예상 동작 

- redux 스토어에서 token 정보를 관리하고, 토큰이 필요한 요청시마다 헤더를 세팅하여 보내줄 것입니다.
- 새로운 accessToken이 발급될 경우 token 정보를 갱신합니다.

## 진행 사항 

- [ ] auth 스토어 수정
- [ ] token 갱신 작업

**Comment by edegiil** [comment]
스토어에서 token을 관리하게 되면 로컬스토리지에서는 아예 없애는 건가요?

**Comment by Seogeurim** [comment]
아니요! 로컬 스토리지에서 없애지 않고 스토어에서 token 정보를 관리해서 api 요청 때 넣어주려고 합니다. 혹시 다른 방안이 있을까요?
![image](https://user-images.githubusercontent.com/22045163/130730175-1ae92b2d-01a6-4a04-ab92-a32070a9646b.png)
![image](https://user-images.githubusercontent.com/22045163/130730186-1f7b4a68-574d-4306-b578-352eab50ab59.png)

**Comment by edegiil** [comment]
아 저렇게 하면 괜찮습니다.. 스토어에만 보관하면 새로고침했을 때 다 날아가서...

**Comment by Seogeurim** [comment]
맞아요 !! ㅎㅎ
